src/main/resources/db/migration/V0001__Base_version.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE public.case_payment_orders (
+                         id integer not null PRIMARY KEY,
```

Comment:
The requirement for this has since changed to a [UUID type](https://www.postgresql.org/docs/9.1/datatype-uuid.html)

Reply:
Oh yes, I was meant to come back to investigate that....thanks !

Reply:
Fixed in commit https://github.com/hmcts/cpo-case-payment-orders-api/pull/16/commits/f47a41148bdbc4123ae3b8761b453b6e8b535f8d